LiskHQ/lisk
# Update loader module in chain to use network module and fix the tests - Closes #2875

### What was the problem?

The loader was interacting with individual peers directly (e.g. for synching).

### How did I fix it?

- The loader module now interacts with the network via the new network module.
- Fixed a few issues related to the network module.

### How to test it?

- Run the functional and integration tests.
- Sync the node with testnet and check that the loader functionality still works.

### Review checklist

* The PR resolves #2875
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
